### PR TITLE
Create a standalone updater

### PR DESCRIPTION
**Pull request by anaisbetts**
This PR adds a new EXE that will push around `UpdateManager` via a simple console EXE. This allows apps that aren't themselves written in .NET to use Squirrel.
## TODO:
- [x] Check for Updates
- [x] Download / Apply updates

### CODE REVIEW

File: src/Squirrel.Tests/Squirrel.Tests.csproj [hunk 1]
```diff
@@ -233,7 +233,9 @@
       <Name>Squirrel.WiXUi</Name>
     </ProjectReference>
   </ItemGroup>
-  <ItemGroup />
+  <ItemGroup>
+    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
```
Comment (shiftkey) [inline src/Squirrel.Tests/Squirrel.Tests.csproj:237]: yay, VS2013!

File: src/Squirrel.Client/UpdateManager.cs [hunk 2]
```diff
@@ -448,7 +456,7 @@ void checksumPackage(ReleaseEntry downloadedRelease)
 
         List<string> installPackageToAppDir(UpdateInfo updateInfo, ReleaseEntry release)
         {
-            var pkg = new ZipPackage(Path.Combine(rootAppDirectory, "packages", release.Filename));
+            var pkg = new ZipPackage(Path.Combine(updateInfo.PackageDirectory, release.Filename));
```
Comment (shiftkey) [inline src/Squirrel.Client/UpdateManager.cs:459]: All of the :sparkles: for killing off the hard-coded strings

File: src/Squirrel.Updater/Program.cs [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+                    var releaseNotes = new Dictionary<ReleaseEntry, string>();
+
+                    try {
+                        releaseNotes = (updateInfo.ReleasesToApply.Count > 0) ? updateInfo.FetchReleaseNotes() : releaseNotes;
+                    } catch (Exception ex) {
+                        // TODO: Find a way to log this
+                    }
+
+                    Console.WriteLine(JsonConvert.SerializeObject(new {
```
Comment (shiftkey) [inline src/Squirrel.Updater/Program.cs:83]: Can we do this in a non-JSON format if it's just for the console output?

**Comment by shiftkey** [comment]
Thanks for tidying up and adding the serialization info. 

Looks good, let me know if there's anything specific I need to dig deeper into.

File: src/Squirrel.Updater/Program.cs (hunk 3 above)
Comment (anaisbetts) [inline src/Squirrel.Updater/Program.cs:83]: The idea is that it's not human-readable output, it's going to be parsed by a parent application exec'ing this app

Comment (shiftkey) [inline src/Squirrel.Updater/Program.cs:83]: :thumbsup:

**Pull request merged**